### ISSUE TITLE
Qfield for windows does not start

### ISSUE THREAD

**Issue by Fertuesta25** [opengisch/QField#3121]
### Describe the issue
When you want to open qfield in windows it does not start

### Desktop (please complete the following information)
 - OS: [e.g. Windows]
 - QGIS Version [e.g. 3.26.0]
 - QFieldSync Version [e.g.4.0.0]

**Comment by m-kuhn** [comment]
Which QField version did you install? Did it work before?

**Comment by Fertuesta25** [comment]
2.2.1 - Coordinated Capybara, the qfield-v2.0.14-windows-x64.exe version did work.

**Comment by m-kuhn** [comment]
Indeed, this one is broken and we forgot to remove it from the release page. All official download links should point to 2.0.14 already and this one is now removed from the release page too.
Thank you for reporting.

**Comment by m-kuhn** [comment]
Thanks for the heads up

```
qrc:/qml/qgismobileapp.qml:938:3: Type DashBoard unavailable
qrc:/qml/DashBoard.qml:334:3: Type TemporalProperties unavailable
qrc:/qml/TemporalProperties.qml:155:5: Type CalendarPanel unavailable
qrc:/qml/imports/QFieldControls/CalendarPanel.qml:4:1: module "Qt.labs.calendar" is not installed
```

**Comment by Fertuesta25** [comment]
Excellent this problem with windows was solved in version 2.3.1 of QField, thanks. It would be nice if the font size in the QField for windows user interface is similar to the default font size of the operating system.

Currently the font size of the QField user interface is somewhat large.
![image](https://user-images.githubusercontent.com/59838508/188280511-f23be4ad-ef37-42e0-a16c-254b2c284ea3.png)
